server/tests/audit_math/test_minerva.py

```diff
@@ -88,6 +88,24 @@ def test_get_sample_size_landslide():
     }
 
 
+def test_get_sample_size_landslide_third_party():
+    c = minerva.make_arlo_contest({"a": 1000, "b": 900, "c": 0})
+    res = minerva.get_sample_size(10, c, None, None)
```

Comment:
Nitpick: it'd be nice to use a variable name here instead of just "10" for the risk limit. (If you wanted to drop a `RISK_LIMIT = 10` in global scope and just use that variable here that would be great!)